Apps/US/DynamicsGPMigrationVendor1099/app/src/Upgrade/GPVendor1099MigInstall.Codeunit.al

```diff
@@ -0,0 +1,66 @@
+codeunit 42000 "GP Vendor 1099 Mig. Install"
+{
+    Subtype = Install;
+
+    var
+        GPVendor1099MappingHelpers: Codeunit "GP Vendor 1099 Mapping Helpers";
+
+    trigger OnInstallAppPerDatabase()
+    begin
+        GPVendor1099MappingHelpers.CleanMappings();
+        Install2022Mappings();
+    end;
+
+    local procedure Install2022Mappings()
+    var
+        TaxYear: Integer;
+    begin
+        TaxYear := 2022;
```

Comment:
is this something which requires yearly updates to the extension (2023, 2024, etc.)

Reply:
Yes, it will require maintenance each year if the 1099 format changes.